scripts/build_powerplants.py

```diff
@@ -320,7 +320,7 @@ def replace_natural_gas_technology(df: pd.DataFrame):
         if "EXTERNAL_DATABASE" not in config:
             logger.error(
                 "Missing configuration EXTERNAL_DATABASE in powerplantmatching config yaml\n\t"
-                "Please check file configs/powerplantmatching_config.yaml"
+                "Please check file configs/gp/powerplantmatching_config.yaml"
```

Comment:
why "gp"? why don't keep them in the same folder?

Reply:
There are configuration files which shouldn't be changed by user (e.g. parameters of the powerplantmatching). To facilitate navigation, it feels handy to distinguish those files from the configs which are relevant for modelling. `gp` stands for `generic product` but we can pick any other name 🙂